app/src/test/java/openfoodfacts/github/scrachx/openfood/network/OpenFoodAPIServiceTest.java

```diff
@@ -185,23 +191,30 @@ public void getProduct_notFound() throws Exception {
     @Test
     public void post_product() throws IOException {
         SendProduct product = new SendProduct();
+
         product.setBarcode("978020137962");
         product.setName("coca3");
         product.setBrands("auchan");
         product.setWeight("300");
         product.setWeight_unit("g");
         product.setLang("fr");
```

Comment:
Is this needed anymore except the barcode? We can set everything inside the `productDetails` Map directly.

Reply:
The only useful call would be `product.getQuantity()` on line 205.

Reply:
Can we use Brand neutral values like "Product name" or "OFF brand" ?